app/views/users/_user.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= content_tag :tr, id: dom_id(user), class: "user" do %>
+  <td><%= user.email %></td>
```

Comment:
What do you think about adding a `Resend activation` link here as well? Its very common that people lose this kind of emails, and didn't click the link yet.